composer.json

```diff
@@ -4,11 +4,15 @@
     "type": "project",
     "description": "Symfony Demo Application",
     "autoload": {
-        "psr-4": { "": "src/" },
+        "psr-4": {
```

Comment:
But only if you create a ton of bundles in your app ... which by the way is now considered a bad practice in most of the cases!